src/status_im/contexts/wallet/edit_account/view.cljs

```diff
@@ -51,24 +51,26 @@
                                               :updated-key :name
                                               :new-value   @edited-account-name}))]
     (fn []
-      (let [{:keys [name emoji address color watch-only?]
-             :as   account}         (rf/sub [:wallet/current-viewing-account])
-            network-details         (rf/sub [:wallet/network-preference-details])
-            test-networks-enabled?  (rf/sub [:profile/test-networks-enabled?])
-            network-preferences-key (if test-networks-enabled?
-                                      :test-preferred-chain-ids
-                                      :prod-preferred-chain-ids)
-            account-name            (or @edited-account-name name)
-            button-disabled?        (or (nil? @edited-account-name)
-                                        (= name @edited-account-name))]
+      (let [{:keys            [name emoji address color watch-only?]
+             default-account? :wallet
+             :as              account} (rf/sub [:wallet/current-viewing-account])
```

Comment:
this destructuring will need to be updated if we add the key previously mentioned. Just pointing it as a reminder